### DISCUSSION TITLE
Eufy Stream with Eufy Security Add On and WebRTC without RTSP supported camera

### DISCUSSION THREAD

**Discussion by digitalmit** [fuatakgun/eufy_security#1082] [Q&A]
Dear Community, I am trying to figure out whether it is possible in any way to make it work with cameras that are not using RTSP as protocol like the following ones:

IndoorCam S350 (tilt) 4K
Eufy Cam E330 Pro
SoloCam S340 3K dual cam
Eufy E340 Floodlight
Doorbell Eufy E340

My setup is all done with camera connected to 3 different HomeBase3 devices without RTSP protocol enabled or even enablable.

When i setup the plugin, the integration and add on i see all sensors and all different information available in my dashboard but no info for the live streaming. When I try to add the cards on the WebRTC through P2P is gives me error.

Sorry If this is a stupid question, i have been googling for 3 days but haven't found any solution. My doubt is that the only way is with RTSP enabled? However, I think there should be a solution since if you use Google Home you are able to see multiple camera at once in live streaming.

I much appreciate for your help

**Comment by InstigatorX** [comment]
+1

**Reply by gerbenhwk79** [reply]
Some, here using Eufy Cam E330 Pro with a Homebase 3. 

For the camera 3 entities are created (binary_sensor.achtertuin_debug_device, button.achtertuin_reset_alarm, button.achtertuin_trigger_alarm) but no Camera entity

**Reply by Araxion** [reply]
I got camera entity, but showing only static image

**Reply by komtx** [reply]
Same here.  I have Go2RTC installed as I was told it's less resource intensive and I always get HLS Streams Unknown error
